### PR TITLE
update typos and gramar for FR

### PR DESCRIPTION
**Pull request by satblip**
I have update french lang file to fix typos, gramar and add few translations

### CODE REVIEW

File: lang/lang_en_fr.txt [hunk 1]
```diff
@@ -304,11 +304,11 @@
 
 #MSG_SELFTEST_EXTRUDER_FAN_SPEED c=18 r=0
 "Extruder fan:"
-"Ventilo extrudeur:"
+"Ventilation extrudeur:"
```
Comment (3d-gussner) [inline lang/lang_en_fr.txt:307]: This is too long as shown above c=18 r=0 means max. 18 characters and one row

File: lang/lang_en_fr.txt [hunk 2]
```diff
@@ -348,7 +348,7 @@
 
 #MSG_RESPONSE_POOR c=20 r=2
 "Fil. sensor response is poor, disable it?"
-"La reponse du capteur de fil. est pauvre, le desactiver ?"
+"La reponse du capteur de filament est pauvre, le desactiver ?"
```
Comment (3d-gussner) [inline lang/lang_en_fr.txt:351]: This is too long as shown above c=20 r=2 means max. 20 characters per row and max 2 rows. This translation will use 3 rows

File: lang/lang_en_fr.txt [hunk 3]
```diff
@@ -448,19 +448,19 @@
 
 #MSG_FILAMENTCHANGE c=0 r=0
 "Change filament"
-"Changer filament"
+"Changer le filament"
```
Comment (3d-gussner) [inline lang/lang_en_fr.txt:451]: This might not fit on the LCD screen, did you compile and test this change?

File: lang/lang_en_fr.txt [hunk 4]
```diff
@@ -448,19 +448,19 @@
...
 
 #MSG_CHANGE_SUCCESS c=0 r=0
 "Change success!"
 "Changement reussi!"
 
 #MSG_CORRECTLY c=20 r=0
 "Changed correctly?"
-"Change correctement?"
+"Le changement est correct ?"
```
Comment (3d-gussner) [inline lang/lang_en_fr.txt:459]: too long, see above

File: lang/lang_en_fr.txt [hunk 5]
```diff
@@ -448,19 +448,19 @@
...
 
 #MSG_CHANGING_FILAMENT c=20 r=0
 "Changing filament!"
-"Changement filament!"
+"Changement du filament!"
```
Comment (3d-gussner) [inline lang/lang_en_fr.txt:463]: too long, see above

File: lang/lang_en_fr.txt [hunk 6]
```diff
@@ -572,7 +572,7 @@
 
 #MSG_SELFTEST_EXTRUDER_FAN c=20 r=0
 "Left hotend fan?"
-"Ventilo tete gauche?"
+"Ventilateur tete gauche?"
```
Comment (3d-gussner) [inline lang/lang_en_fr.txt:575]: will use two rows, see above

File: lang/lang_en_fr.txt [hunk 7]
```diff
@@ -1204,28 +1204,36 @@
...
 "XYZ calibration compromised. Left front calibration point not reachable."
+"La calibration XYZ est defectueuse. Les points de calibration de l'avant-gauche sont inatteignables."
 
 #MSG_UNLOAD_ALL c=17 r=0
 "Unload all"
+"Tout retirer"
 
 #MSG_UNLOAD_FILAMENT_1 c=17 r=0
 "Unload filament 1"
+"Retirer le filament 1"
```
Comment (3d-gussner) [inline lang/lang_en_fr.txt:1239]: too long, see above

**Review by 3d-gussner — CHANGES_REQUESTED** [review]
You need to consider the max. characters to be used and the rows.
The display will split the message into rows at the word that hits the limit.

Some messages are limited to c=17 (smaller than c=20) due to multiple information on the LCD or additional values behind it. If you expend that limit this may be unreadable.

Just a tip, before you post language PRs it is good if you compile your changes and review them on your printer LCD. This way the chances are higher to get it merged.

**Comment by satblip** [comment]
Ok @3d-gussner thanks for remarks, I'll change them soon to fit the needs

**Comment by 3d-gussner** [comment]
@satblip you should check all messages for these limits. Also compare it to the latest version from Prusa. As said if you compile it and test your changes on your printer the chances are higher to get your pull request pulled.
I had some in the past bit too long messages and they messed up the results on the LCD :-) also splitting words may help to make long messages easier to read, don't know if that is a common thing in French but in German splitting long words is quite often used in newspapers to fit the columns they are restricted to.
I hope I learned my lesson.

**Comment by 3d-gussner** [comment]
@satblip I think these have been fixed, can you review your PR and if it is not needed anymore please close it.

**Comment by satblip** [comment]
@3d-gussner indeed that has moved forward, thanks for the follow up!

**Pull request closed without merging**